README.md

```diff
@@ -50,9 +50,9 @@ $default = function (RequestInterface $request) {
     return new Response();
 };
 
-$stack = new Stack($middleware, $default);
+$stack = new Stack(...$middleware);
```

Comment:
I'm not really sure about this change. What does splat buy us here? From a syntax standpoint, it is more to type out. From a functionality standpoint, all it does it splat and unsplat the arguments, which consumes ticks unnecessarily.